src/detext/run_detext.py

```diff
@@ -122,7 +122,8 @@ class Args(NamedTuple):
     init_weight: float = 0.1  # weight initialization value.
     pmetric: str = None  # Primary metric.
     all_metrics: List[str] = None  # All metrics.
-    score_rescale: List[float] = None  # The mean and std of previous model. For score rescaling, the score_rescale has the xgboost mean and std.
+    score_rescale: List[
+        float] = None  # The mean and std of previous model. For score rescaling, the score_rescale has the xgboost mean and std.
```

Comment:
is this formatting intentional?

Reply:
Oh good catch. Seems to be automatically formatted by IDE. Fixed.